wasm/elastix-wasm.cxx

```diff
@@ -54,10 +58,10 @@ class PipelineFunctor
     InputImageType movingImage;
     pipeline.add_option("-m,--moving", movingImage, "Moving image")->type_name("INPUT_IMAGE");
 
-    std::string initialTransformFile;
-    pipeline
-      .add_option("-i,--initial-transform", initialTransformFile, "Initial transform to apply before registration")
-      ->type_name("INPUT_BINARY_FILE");
+    using InputTransformType = itk::wasm::InputTransform<CompositeTransformType>;
+    InputTransformType initialTransform;
+    pipeline.add_option("-i,--initial-transform", initialTransform, "Initial transform to apply before registration")
+      ->type_name("INPUT_TRANSFORM");
```

Comment:
Just wondering, the elastix.exe command-line interface has `"-t0"` for the initial transform of a registration, whereas elastix-wasm now has both `"-i"` and `"-t"` (but not `"-t0"`). Should these command-line flags still be brought in sync, one way or another? Or are they different for a fundamental reason?

Reply:
Yes, we can/should bring them into sync! I will update this one -- please let me know if you see any others that should be updated.